apps/bestofjs-nextjs/src/app/backend.ts

```diff
@@ -352,14 +346,8 @@ async function fetchProjectData(): Promise<{
 function fetchDataFromRemoteJSON() {
   const url = FETCH_ALL_PROJECTS_URL + `/projects.json`;
   console.log(`Fetching JSON data from ${url}`);
-  const options = { next: { revalidate: 60 * 60 } }; // Revalidate in one hour
-  return fetch(url, options).then((res) => res.json());
-}
-
-function fetchDataFromFileSystem() {
-  console.log("Fetch from the file system");
-  const filepath = path.join(process.cwd(), "public", "data/projects.json");
-  return fs.readJSON(filepath);
+  // const options = { next: { revalidate: 60 * 60 } }; // Revalidate in one hour
```

Comment:
Why did we remove revalidate here and
[here](
https://github.com/bestofjs/bestofjs/pull/192/files#diff-d55bd9b8687cc88e8524c3ad2be17f7251d45491f5ebc061d2d227eef3a7899aL36)?

Reply:
The comment said:

> "TODO remove when page rebuilt is triggered automatically"

It's exactly what we are doing now, the site is rebuilt everyday on Vercel, just after the static API (the JSON file) has been rebuilt, so we don't need to tell Next.js to revalidate the page. The page is correct until the site is rebuilt, every 24 hours.